src/app/board/quest/quest.component.ts

```diff
@@ -8,4 +9,10 @@ import { IQuest } from '../../shared/quest';
 })
 export class QuestComponent {
   @Input() quest: IQuest;
+  constructor(private questService: QuestService) {}
+
+  public acceptQuest() {
+    this.quest.isAccepted = true;
+    this.questService.updateQuest(this.quest).subscribe();
```

Comment:
If you don't need subscription in component, you can subscribe in service

Reply:
okay, got it, thank you